### PR TITLE
Added Test Suite for cautils

### PR DESCRIPTION
**Pull request by VaibhavMalik4187**

### CODE REVIEW

**Comment by VaibhavMalik4187** [comment]
@amirmalka @matthyx could you please review this PR whenever you have a moment? Thanks!

File: core/cautils/display_test.go [hunk 1]
```diff
@@ -30,3 +33,434 @@ func TestStartSpinner(t *testing.T) {
...
+
+func TestFailureDisplay(t *testing.T) {
+	tests := []struct {
+		text string
+		want string
+	}{
+		{
+			text: "Test",
+			want: "Test",
+			// want: "\033[1;31mTest\033[0m",
```
Comment (matthyx) [inline core/cautils/display_test.go:45]: why did you drop the colors in all similar tests?

Comment (VaibhavMalik4187) [reply core/cautils/display_test.go:45]: I should've highlighted this point in the code itself. The output remains colourless all the time. At first, I thought that there was a bug in the code being used to capture the output. But, I tried directly printing using fmt and it was able to capture all the colors precisely. Which indicates that the outputs of all the display functions are colourless.

Comment (matthyx) [reply core/cautils/display_test.go:45]: ok, so pls remove these comments

Comment (VaibhavMalik4187) [reply core/cautils/display_test.go]: Done

**Review by matthyx — APPROVED** [review]

**Comment by matthyx** [comment]
thanks!

**Pull request merged**